.travis.yml

```diff
@@ -1,15 +1,16 @@
 # Configuration script for Travis-CI
 language: python
+dist: xenial
 python:
   - 2.7
+  - 3.6
+  - 3.7
```

Comment:
Purely out of curiosity: no 3.5 here because there are actual failures/problems somewhere, or merely because you don't care about 3.5 and older versions?

Reply:
I don't care, locally it runs.